net/libfab/libfab.c

```diff
@@ -760,55 +533,46 @@ static void libfab_straddr_gen(struct m0_fab__conn_data *cd, char *buf,
 static uint32_t libfab_handle_connect_request_events(struct m0_fab__tm *tm)
 {
 	struct m0_fab__ep        *ep = NULL;
-	struct m0_fab__conn_data *cd;
-	struct m0_fab__ep_name    en = {};
 	struct fid_eq            *eq;
 	struct fi_eq_err_entry    eq_err = {};
 	struct fi_eq_cm_entry    *cm_entry;
 	char                      entry[(sizeof(struct fi_eq_cm_entry) +
 					sizeof(struct m0_fab__conn_data))];
 	uint32_t                  event;
 	int                       rc;
-	char                      straddr[LIBFAB_ADDR_STRLEN_MAX] = {};
-	int                       ret;
+	struct m0_net_ip_addr     addr;
 
 	eq = tm->ftm_pep->fep_listen->pep_res.fpr_eq;
-	do {
```

Comment:
Need to add back do while from main branch as part of CPU fix

Reply:
Done.